src/schemas/fairdata-ida.ui.js

```diff
@@ -668,6 +684,7 @@ export default {
 		},
 		'placeholder': "– choose license –",
 		'title': "license",
+		'required': () => true,
```

Comment:
why function and why thick pointer?

Reply:
Function so that the code using this can always trust the value to be function. Usually required values that do not need any logic should have the required value in schema, not in UI schema. This case is special because metax doesn't require it. What do you mean with thick pointer?

Reply:
the fat arrow / thick pointer binds the scope, so that does look odd in this case.

Reply:
There is no need to bind the context in this case but doesn't do any harm. It is written in this manner just to make it shorter and cleaner.

Reply:
you misunderstood